### DISCUSSION TITLE
`latest` tag is being overriden from uplink/npm - is this right?

### DISCUSSION THREAD

**Discussion by seanlandsman** [verdaccio/verdaccio#5009] [General]
Hi,

We publish versions of modules to Verdaccio and so far we've updated the `latest` tag with `dist-add` to Verdaccio and up until today this has worked fine.

We've published a version of the module(s) to npm today and then later (3 hrs later) another version to verdaccio and updated the `latest` tag to this new version (on Verdaccio, not npm). What we're seeing now is that after a while the `latest` tag on Vedaccio changes to whats on npm.

Is this right? We've used the above mechanism for ages and it worked fine - this seems to be a change in behaviour that we can't explain.

I searched for similar discussions but couldn't find anything like this.

thanks

**Comment by mbtools** [comment]
It's tricky. See this issue https://github.com/verdaccio/verdaccio/issues/3913 and my [comment](https://github.com/verdaccio/verdaccio/issues/3913#issuecomment-2699352307).

Best to post an example with version numbers and tags to get more help.
